docs/uppmax.md

```diff
@@ -74,14 +76,170 @@ It is not suitable for use with real data.
+$ sftp <USER>-<BIANCA_PROJECT>@bianca-sftp.uppmax.uu.se:<USER>-<BIANCA_PROJECT>
+
+# Transfer nextflow to the wharf area
+sftp> put nextflow-<NEXTFLOW_VERSION>-all .
+
+# Exit sftp
+$ exit
+
+# Connect to bianca
+$ ssh -A <USER>-<BIANCA_PROJECT>@bianca.uppmax.uu.se
+
+# Go to your project
+$ cd /castor/project/proj_nobackup
+
+# Make folder for Nextflow
+$ mkdir tools
+$ mkdir tools/nextflow
+
+# Move Nextflow from the wharf area to its directory
+$ mv /castor/project/proj_nobackup/wharf/<USER>/<USER>-<BIANCA_PROJECT>/nextflow-<NEXTFLOW_VERSION>-all /castor/project/proj_nobackup/tools/nextflow
 
-For security reasons, there is no internet access on Bianca so you can't download from or upload files to the cluster directly. Before running a nf-core pipeline on Bianca you will first have to download the pipeline and singularity images needed elsewhere and transfer them via the wharf area to your Bianca project.
+# Establish permission
+$ chmod a+x /castor/project/proj_nobackup/tools/nextflow/nextflow-<NEXTFLOW_VERSION>-all
 
-You can follow the guide for downloading pipelines [for offline use](https://nf-co.re/tools#downloading-pipelines-for-offline-use). Note that you will have to download the singularity images as well.
+# If you want other people to use it
+# Be sure that your group has rights to the directory as well
+$ chown -R .<BIANCA_PROJECT> /castor/project/proj_nobackup/tools/nextflow/nextflow-<NEXTFLOW_VERSION>-all
+
+# Make a link to it
+$ ln -s /castor/project/proj_nobackup/tools/nextflow/nextflow-<NEXTFLOW_VERSION>-all /castor/project/proj_nobackup/tools/nextflow/nextflow
+
+# And every time you're launching Nextflow, don't forget to export the following ENV variables
+# Or add them to your .bashrc file
+$ export NXF_HOME=/castor/project/proj/nobackup/tools/nextflow/
+$ export PATH=${NXF_HOME}:${PATH}
+$ export NXF_TEMP=$SNIC_TMP
+$ export NXF_LAUNCHER=$SNIC_TMP
+$ export NXF_SINGULARITY_CACHEDIR=/castor/project/proj_nobackup/singularity-images
+```
+
+### Install nf-core tools
+
+```bash
+# Connect to rackham
+$ ssh -X <USER>@rackham.uppmax.uu.se
+# Or stay in your terminal
+
+# Install the latest pip version
+$ pip3 install --upgrade --force-reinstall git+https://github.com/nf-core/tools.git@dev --user
+```
 
-Next transfer the pipeline and the singularity images to your project. Before running the pipeline you will have to indicate to nextflow where the singularity images are located by setting `NXF_SINGULARITY_CACHEDIR` :
+### Download and transfer a nf-core pipeline
 
-`export NXF_SINGULARITY_CACHEDIR=Your_Location_For_The_Singularity_directory/.`
+```bash
+# Connect to rackham
+$ ssh -X <USER>@rackham.uppmax.uu.se
+# Or stay in your terminal
+
+# Open an interactive session (if you are on rackham)
+$ interactive <rackham_project>
+
+# Download a pipeline (nf-core/rnaseq 3.0) with the singularity images
+$ nf-core download rnaseq -r 3.0 -s --compress none
+
+# If necessary, extra singularity images can be download separately
```

Comment:
I think this is _only_ Sarek. What do you think about leaving this out and keeping it just the Sarek specific docs?

Reply:
It's just these 4 lines, it would definitively make senses to have that in a sarek specific doc, I agree, but it's more convenient to have all that here in the same docs